firestore/integration_test_internal/src/firestore_integration_test.cc

```diff
@@ -1,6 +1,7 @@
 // Copyright 2021 Google LLC
 
 #include "firestore_integration_test.h"
```

Comment:
⚠️ Lint warning: `Include the directory when naming .h files`<hidden value="cpplint-file-comment"></hidden><hidden value="firestore/integration_test_internal/src/firestore_integration_test.cc:3:  Include the directory when naming .h files  [build/include_subdir] [4]"></hidden>